src/umpire/Umpire.cpp

```diff
@@ -98,6 +99,20 @@ bool pointer_contains(void* left_ptr, void* right_ptr)
   }
 }
 
+bool is_accessible(Platform p, Allocator a) 
```

Comment:
I'm not sure how the `camp` namespace is being made available.  But since this is a library function, shouldn't the camp namespace be explicitly stated?
```suggestion
bool is_accessible(camp::Platform p, Allocator a) 
```

Reply:
We bring the Platform enum into the umpire namespace too.